prismic-model/js/parts/article-body.js

```diff
@@ -45,6 +45,10 @@ export default {
       }, {
         'name': 'standalone',
         'display': 'Standalone'
+      } ],
+      'quoteV2': [ {
```

Comment:
I'd like to work on this a little more with the content team, as, as far as I could tell the immediate use for this is not pull quotes but reviews?